tokens/src/core/alias/size.json

```diff
@@ -1,15 +1,16 @@
 {
   "size": {
+    "$type": "dimension",
     "border": {
-      "width": { "value": "1px", "type": "dimension", "source": "$border-width", "description": "Default border width." },
+      "width": { "source": "$border-width", "$value": "1px", "$description": "Default border width." },
```

Comment:
[curious] Cross-checking the `dimension` type with the DTCG spec again, I noticed that it seems you can also define the dimension value and its unit separately, e.g.:

```json
{
  "size": {
    "$type": "dimension",
    "border": {
      "width": {
        "$value": {
          "value": 1,
          "unit": "px"
        }
      }
     }
  }
}
```

Not sure if this is something `style-dictionary` v4 supports out-of-the-box. Do we think it's worth exploring defining dimension token values and units separately vs. combined as we have it today (i.e., `"$value": "1px"`).

Reply:
Since this is what is documented in the DTCG spec I'd think we'd want to move forward with this change. Let's verify this works with style-dictionary v4 and figure out next steps from there.

Reply:
For posterity, we tried to update the token definition to adopt the DTCG format spec for `$dimension` tokens, but it seems to result in unexpected output, e.g.:

```css
--pgn-size-border-width-value: 1; /* Default border width. */
--pgn-size-border-width-unit: 0px; /* Default border width. */
```

When we'd really want something like:

```css
--pgn-size-border-width-value: 1px; /* Default border width. */
```

I've opened an issue upstream in the `style-dictionary` repository to try to drive clarity in the expected output for `$dimension` tokens using the DTCG spec / report a bug, if applicable: https://github.com/amzn/style-dictionary/issues/1398

Given this, for now, we will keep the Paragon `$dimension` tokens using the combined `$value` vs. following the DTCG spec.